t/t1001-mf-priority.t

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+test_description='Test multi-factor priority plugin'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+# BULK_UPDATE=${FLUX_BUILD_DIR}/t/scripts/send_fake_payloads.py
+
+export TEST_UNDER_FLUX_NO_JOB_EXEC=y
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 1 job
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'load multi-factor priority plugin' '
+        flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
+'
+
+test_expect_success 'list all loaded plugins' '
+        flux jobtap list | grep mf_priority
+'
+
+test_expect_success 'try to submit a job when user does not exist in DB' '
+        test_must_fail flux mini submit -n1 hostname > failure.out 2>&1 &&
+        test_debug "cat failure.out" &&
+        grep "user not found in flux-accounting DB" failure.out
+'
+
+test_expect_success 'send an empty payload to make sure unpack fails' '
+cat <<-EOF >bad_payload.py
+import flux
+
+#create a JSON payload
+flux.Flux().rpc("job-manager.mf_priority.rec_update", {}).get()
+EOF
+flux python bad_payload.py
```

Comment:
Check that `bad_payload.py` fails with `test_must_fail`.

Also, missing `&&` between these two commands. (You might try running tests with `--chain-lint` to see if there are any other missing `&&`-chains.)


```suggestion
test_must_fail flux python bad_payload.py &&
```